plugins/geospatial/server/geospatial.py

```diff
@@ -492,10 +492,10 @@ def setGeospatial(self, item, params):
         .param('id', 'The ID of the item.', paramType='path')
         .param('body', 'A JSON object containing the geospatial fields to add.',
                paramType='body')
-        .errorResponse('ID was invalid.')
-        .errorResponse('Invalid JSON was passed in request body.')
-        .errorResponse('Geospatial key name was invalid.')
-        .errorResponse('Geospatial field did not contain valid GeoJSON.')
+        .errorResponse('ID was invalid.\n\n'
+                       'Invalid JSON was passed in request body.\n\n'
+                       'Geospatial key name was invalid.\n\n'
+                       'Geospatial field did not contain valid GeoJSON.')
```

Comment:
We could allow `errorResponse` to accept a list as the first parameter, and then inspect the type and call `'\n\n'.join(val)`.